sass/application.scss

```diff
@@ -76,6 +76,11 @@ $govuk-page-width: 1170px;
   text-align: right;
 }
 
+.right-content {
```

Comment:
This was originally in header.scss so just moved it here for now.  Not a fan of this class but couldn't delete as it's used elsewhere at the moment.